pkg/daemon/daemon.go

```diff
@@ -685,6 +662,63 @@ func (dn *Daemon) nodeStateSyncHandler() error {
 	return nil
 }
 
+func (dn *Daemon) shouldSkipReconciliation(latestState *sriovnetworkv1.SriovNetworkNodeState) (bool, error) {
```

Comment:
perhaps extend daemon UT with some test cases for this.

e.g when plugin reports that status changed ?